### ISSUE TITLE
Blackstone Shales has been McOopsied

### ISSUE THREAD

**Issue by ghost** [TerraformersMC/Cinderscapes#81]
```
[22:00:30] [main/ERROR] (Minecraft) Feature: Not a JSON object: "minecraft:ore_debris_small"; Not a JSON object: "minecraft:ore_debris_large"; Not a JSON object: "minecraft:ore_quartz_deltas"; Not a JSON object: "minecraft:ore_gold_deltas"; Not a JSON object: "minecraft:spring_closed_double"; Not a JSON object: "minecraft:ore_magma"; Not a JSON object: "minecraft:red_mushroom_nether"; Not a JSON object: "minecraft:brown_mushroom_nether"; Not a JSON object: "minecraft:glowstone"; Not a JSON object: "minecraft:glowstone_extra"; Not a JSON object: "minecraft:patch_soul_fire"; Not a JSON object: "minecraft:patch_fire"; Not a JSON object: "minecraft:spring_delta"; Not a JSON object: "cinderscapes:shales"; Not a JSON object: "cinderscapes:gravel_blackstone_shales"; Not a JSON object: "cinderscapes:soul_soil_blackstone_shales"; Not a JSON object: "cinderscapes:soul_sand_blackstone_shales"
[22:00:30] [main/ERROR] (Minecraft) Feature: Not a JSON object: "minecraft:spring_lava_double"
[22:00:30] [main/ERROR] (Minecraft) Feature: Not a JSON object: "minecraft:ore_debris_small"; Not a JSON object: "minecraft:ore_debris_large"; Not a JSON object: "minecraft:ore_quartz_deltas"; Not a JSON object: "minecraft:ore_gold_deltas"; Not a JSON object: "minecraft:spring_closed_double"; Not a JSON object: "minecraft:ore_magma"; Not a JSON object: "minecraft:red_mushroom_nether"; Not a JSON object: "minecraft:brown_mushroom_nether"; Not a JSON object: "minecraft:glowstone"; Not a JSON object: "minecraft:glowstone_extra"; Not a JSON object: "minecraft:patch_soul_fire"; Not a JSON object: "minecraft:patch_fire"; Not a JSON object: "minecraft:spring_delta"; Not a JSON object: "cinderscapes:shales"; Not a JSON object: "cinderscapes:gravel_blackstone_shales"; Not a JSON object: "cinderscapes:soul_soil_blackstone_shales"; Not a JSON object: "cinderscapes:soul_sand_blackstone_shales"
[22:00:30] [main/ERROR] (Minecraft) Feature: Not a JSON object: "minecraft:spring_lava_double"
```

Idk, it just started doing this and none of the features in the Blackstone Shales are generating. Everything else is working fine except for that. @shartte any ideas?

**Comment by shartte** [comment]
Are you testing this ONLY with Cinderscapes?

**Comment by ghost** [comment]
Yes it's just in a normal dev env with only Cinderscapes. I'm guessing something is configured incorrectly but I touched like literally nothing and now it's upset with me.

**Comment by shartte** [comment]
Uh you closed it, whats going on? This error usually occurs if _something_ is unregistered. There's an audit mod by TelepathicGrunt that can help with figuring out what.

**Comment by ghost** [comment]
Somehow we got it working again, we just released Cinderscapes for 1.16.2, thanks for all of your help on it. I'll have to make sure to check out that audit mod and maybe add it to our workspace for next time.
